### ISSUE TITLE
how do I render unicode utf-8 characters?

### ISSUE THREAD

**Issue by arxii** [developit/preact#497]
`
h("div",false,"&bull;")
h("div",false,"•")
`

Hey guys, I tried to figure out the issue of rendering special characters...
escapes the "&" to render "&amp;bull;" and not •. Am I doing something wrong? how do i unescape?

**Comment by arxii** [comment]
seems like using `\u2022` instead works.

**Comment by developit** [comment]
Thanks for posting your findings so others see them! 👌

**Comment by ATempsch** [comment]
I've got some additional findings to this, hopefully that saves someone a couple hours.

If you've got a string constant with some utf-8 character like the german umlauts "äöü" this will render perfectly fine in a utf-8 encoded page. As soon as you try to load your App into e.g. ISO-8859-1 page your utf-8 characters will be first converted to a html-entity and be rendered as described above with entity double-escaped.

**tl;dr:** In a ISO-8859-1 encoded website the string "Händler" is rendered as "H&amp;auml;ndler".

The solution for now was to follow the instruction above and use "\u00E4" instead of "ä" in my constant.

**Comment by marvinhagemeister** [comment]
@RomanArminTech just checked against the code in `master` and using any utf8 char works fine for me.
